AllReadyApp/Web-App/AllReady/wwwroot/js/eventDetails.js

```diff
@@ -2,107 +2,107 @@
 (function (HTBox) {
     var EventDetailAdmin = (function () {
         function EventDetailAdmin() {
-            $("#messageVolunteersModal").on("show.bs.modal", function (e) {
+            $('#messageVolunteersModal').on('show.bs.modal', function (e) {
```

Comment:
The changes to this file and taskDetails.js seem to be unrelated to the pull request